src/IsExternalInit/IsExternalInit.cs

```diff
@@ -36,7 +36,6 @@
 // SOFTWARE.
 #endregion
 
-#if !ISEXTERNALINIT_DISABLE
```

Comment:
Can we keep this constant? Removing it could be a breaking change (e.g. for a project setup which references the package globally in a `Directory.Build.props` and then excludes the attribute in selected sub-projects which only ever targeted .NET >= 5).

Reply:
Migration should be easy as MSBuild is on their side all they got to do is add this to their ``PackageReference`` then:

``Condition="'$(ISEXTERNALINIT_DISABLE)' != '' AND '$(ISEXTERNALINIT_DISABLE)' == 'false'"`` and then it should work like normally. 👍 You then would have to warn them to do that however so then it won't affect them much plus then it does the same and changes it to an msbuild property instead of something they should add to ``DefineConstant``s on the project. However I think a note saying that they would need to change their csproj is ok and minimal change on their end too as long as they know *what* to change.